### PR TITLE
Claims v1 copy

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java [hunk 1]
```diff
@@ -177,22 +145,16 @@
 
     createMessage(
       "message.claims.record.ok",
-      new MessageBodyParagraph("Tack! Det är allt jag behöver just nu"),
+      new MessageBodyParagraph("Tack! Din anmälan kommer nu hanteras"),
       2000);
     addRelay("message.claims.record.ok", "message.claims.record.ok2");
 
+    final String HANDS_EMOJI = "\uD83D\uDE4C";
```
Comment (meletios) [inline src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: should this be in the begging of the file?

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java [hunk 2]
```diff
@@ -177,22 +145,16 @@
...
     createMessage(
       "message.claims.record.ok2",
       new MessageBodyParagraph(
-        "Jag återkommer till dig om jag behöver något mer, eller för att meddela att jag kan betala ut ersättning direkt"),
+        "Jag återkommer här i chatten   om jag behöver något mer eller för att berätta hur det går " + HANDS_EMOJI),
```
Comment (meletios) [inline src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: i think there is an extra space between chatten  and om

Comment (palmfjord) [reply src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: oops

File: src/main/java/com/hedvig/botService/chat/ClaimsConversation.java (hunk 1 above)
Comment (palmfjord) [reply src/main/java/com/hedvig/botService/chat/ClaimsConversation.java]: idk, should it?

**Pull request merged**